netmiko/fortinet/fortinet_ssh.py

```diff
@@ -1,104 +1,298 @@
+
+		:param enter_config_mode: Do you enter config mode before sending config commands
+
+		:param error_pattern: Regular expression pattern to detect config errors in the
+		output.
+
+		:param terminator: Regular expression pattern to use as an alternate terminator in certain
+		situations.
+
+		:param bypass_commands: Regular expression pattern indicating configuration commands
+		where cmd_verify is automatically disabled.
+		"""
+		return super().send_config_set(
+			config_commands=config_commands,
+			exit_config_mode=exit_config_mode,
+			read_timeout=read_timeout,
+			delay_factor=delay_factor,
+			max_loops=max_loops,
+			strip_prompt=strip_prompt,
+			strip_command=strip_command,
+			config_mode_command=config_mode_command,
+			cmd_verify=cmd_verify,
+			enter_config_mode=enter_config_mode,
+			error_pattern=error_pattern,
+			terminator=terminator,
+			bypass_commands=bypass_commands
+		)
+
+	def enable(self, cmd="", pattern="", re_flags=re.IGNORECASE):
+		"""Not Supported for this platform"""
+		pass
+
+	def exit_enable_mode(self, exit_command: str = "") -> str:
+		"""Not Supported for this platform"""
+		pass
+
+	def set_terminal_width(self):
+		"""Not Supported for this platform"""
+		pass
+
+	def check_config_mode(
+		self, 
+		check_string: str = r"(\) #|\) \$)", 
+		pattern: str = r"", 
+		force_regex: bool = True
+		) -> bool:
+		"""Checks if the device is in configuration mode or not.
+
+		:param check_string: Identification of configuration mode from the device
+		:type check_string: str
+
+		:param pattern: Pattern to terminate reading of channel
+		:type pattern: str
+		"""
+		return super().check_config_mode(
+			check_string=check_string, pattern=pattern, force_regex=force_regex
+		)
+
+	def exit_config_mode(self, exit_config="end", pattern=r"(#|\$)"):
+		"""Exit from configuration mode.
+
+		:param exit_config: Command to exit configuration mode
+		:type exit_config: str
+
+		:param pattern: Pattern to terminate reading of channel
+		:type pattern: str
+		"""
+		return super().exit_config_mode(
+			exit_config=exit_config, pattern=pattern
+		)
+
+	def disable_paging(
+		self,
+		command: str = "",
+		delay_factor: Optional[float] = None,
+		cmd_verify: bool = True,
+		pattern: Optional[str] = None,
+		) -> str:
+		"""Disable paging is only available with specific roles so it may fail.
+
+		:param command: Device command to disable pagination of output
+
+		:param delay_factor: Deprecated in Netmiko 4.x. Will be eliminated in Netmiko 5.
+		"""
+
+		disable_paging_commands = [
+			"config global",
+			"config system console",
+			"set output standard",
+			"end"
+			]
+		output = self.send_config_set(
+			config_commands = disable_paging_commands,
+			exit_config_mode = True,
+			read_timeout = None,
+			delay_factor = 1.0,
+			max_loops = 150,
+			enter_config_mode = False,
+		)
+		log.debug("***Window Paging Disabled***")
+		return output
```

Comment:
So I assume this depends on VDOM support? Also what happens here on failure, does an exception get generated?

Reply:
The command "config global" would fail if the device was not in multi-vdom mode, but the remaining commands are ran with no issue.  It doesn't cause any problem so saw no issue to do any kind of predetermination of what vdom mode it was in.  If you don't like that solution, we can do like the former and discover if the device is in multi-vdom mode and have different commands to run depending on that result.

Reply:
Are okay with this approach, letting the command fail for single-vdom devices since it doesn't break anything?

Reply:
So just verifying this--with VDOM support, the below commands all work?

```
			"config global",
			"config system console",
			"set output standard",
			"end"
```

With no VDOM support, the first command fails with an error, but then the remaining three commands work?